packages/story-editor/src/app/highlights/quickActions/mediaPicker.tsx

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo } from '@googleforcreators/react';
+import {
+  getExtensionsFromMimeType,
+  resourceList,
+} from '@googleforcreators/media';
+import type { Resource } from '@googleforcreators/media';
+import { __, sprintf, translateToExclusiveList } from '@googleforcreators/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TRANSCODABLE_MIME_TYPES, useLocalMedia } from '../../media';
+import { useConfig } from '../../config';
+import { useStory } from '../../story';
+
+interface RenderProps {
+  onClick: () => void;
+}
+interface MediaPickerProps {
+  buttonInsertText?: string;
+  cropParams?: boolean;
+  multiple?: boolean;
+  onClose?: () => void;
+  onPermissionError?: () => void;
+  onSelect: (resource: Resource) => void;
+  onSelectErrorMessage?: string;
+  render: (props: RenderProps) => void;
+  title?: string;
+  type: string | string[];
+}
+
+function MediaPicker({ render, ...props }: MediaPickerProps) {
```

Comment:
@swissspidy Wonder if we should just reuse [this](https://github.com/GoogleForCreators/web-stories-wp/blob/b44192a83fc26ff6c6fd68c4f924ed3db44b8871/packages/story-editor/src/components/form/mediaUploadButton.js#L33) here.